python/src/graph_view.rs

```diff
@@ -268,4 +272,58 @@ impl PyGraphView {
             num_edges, num_vertices, earliest_time, latest_time
         )
     }
+
+    /// Extracts the id from the given python vertex
+    ///
+    /// Arguments:
+    ///     id (str or int): The id of the vertex.
+    pub(crate) fn extract_id(&self, id: &PyAny) -> PyResult<InputVertexBox> {
```

Comment:
I don't think you need to import any of this, utils has the extract_vertex_ref function that you can import:

Have a look at a function like has_vertex in graph_view:

  `  pub fn has_vertex(&self, id: &PyAny) -> PyResult<bool> {
        let v = extract_vertex_ref(id)?;
        Ok(self.graph.has_vertex(v))
    }`